packages/docs/src/content/docs/reference/configuration.md

```diff
@@ -9,14 +9,26 @@ configure Knip.
 
 ## JSON Schema
 
-In JSON, you can use the provided JSON Schema:
+A "@schema" field is a URL that you put at the top of your JSON file. This allows you to get red squiggly lines inside of your IDE when you make a typo or provide an otherwise invalid configuration option.
```

Comment:
```suggestion
A "$schema" field is a URL that you put at the top of your JSON file. This allows you to get red squiggly lines inside of your IDE when you make a typo or provide an otherwise invalid configuration option.
```